.github/ISSUE_TEMPLATE/release-tracking-issue.md

```diff
@@ -31,7 +31,7 @@ Note: remove the backticks (``) from the link.
 -->
 
 - [x] Release `[<RELEASE_NUMBER>](https://github.com/ampproject/amphtml/releases/tag/<RELEASE_NUMBER>)` is cut as a new canary release
-- [ ] Release <RELEASE_NUMBER> pushed to dev channel (<CL submit time>)
+- [ ] Release <RELEASE_NUMBER> pushed to experimental channel (<CL submit time>)
```

Comment:
Do you want to make this explicit and say "... to experimental and beta channels"?

Reply:
Done! Also updated similar cases elsewhere in these issue templates.